--- php.ini-development.old	2013-10-21 09:28:50.358253282 +0800
+++ php.ini-development	2013-10-21 09:35:05.001569978 +0800
@@ -373,7 +373,7 @@
 ; threat in any way, but it makes it possible to determine whether you use PHP
 ; on your server or not.
 ; http://php.net/expose-php
-expose_php = On
+expose_php = Off
 
 ;;;;;;;;;;;;;;;;;;;
 ; Resource Limits ;
@@ -582,7 +582,7 @@
 ; empty.
 ; http://php.net/error-log
 ; Example:
-;error_log = php_errors.log
+error_log = /var/log/php/php-errors.log
 ; Log errors to syslog (Event Log on NT, not valid in Windows 95).
 ;error_log = syslog
 
@@ -703,7 +703,7 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;
 
 ; UNIX: "/path1:/path2"
-;include_path = ".:/php/includes"
+include_path = "."
 ;
 ; Windows: "\path1;\path2"
 ;include_path = ".;c:\php\includes"
@@ -726,7 +726,7 @@
 
 ; Directory in which the loadable extensions (modules) reside.
 ; http://php.net/extension-dir
-; extension_dir = "./"
+extension_dir = /usr/local/lib/php/modules
 ; On windows:
 ; extension_dir = "ext"
 
@@ -756,7 +756,7 @@
 ; will look for to know it is OK to continue execution.  Setting this variable MAY
 ; cause security issues, KNOW WHAT YOU ARE DOING FIRST.
 ; http://php.net/cgi.redirect-status-env
-;cgi.redirect_status_env = 
+;cgi.redirect_status_env =
 
 ; cgi.fix_pathinfo provides *real* PATH_INFO/PATH_TRANSLATED support for CGI.  PHP's
 ; previous behaviour was to set PATH_TRANSLATED to SCRIPT_FILENAME, and to not grok
@@ -765,7 +765,7 @@
 ; of zero causes PHP to behave as before.  Default is 1.  You should fix your scripts
 ; to use SCRIPT_FILENAME rather than PATH_TRANSLATED.
 ; http://php.net/cgi.fix-pathinfo
-;cgi.fix_pathinfo=1
+cgi.fix_pathinfo=0
 
 ; FastCGI under IIS (on WINNT based OS) supports the ability to impersonate
 ; security tokens of the calling client.  This allows IIS to define the
@@ -798,11 +798,11 @@
 ; Temporary directory for HTTP uploaded files (will use system default if not
 ; specified).
 ; http://php.net/upload-tmp-dir
-;upload_tmp_dir =
+upload_tmp_dir = /var/lib/php/upload
 
 ; Maximum allowed size for uploaded files.
 ; http://php.net/upload-max-filesize
-upload_max_filesize = 2M
+upload_max_filesize = 4M
 
 ; Maximum number of files that can be uploaded via a single request
 max_file_uploads = 20
@@ -813,7 +813,7 @@
 
 ; Whether to allow the treatment of URLs (like http:// or ftp://) as files.
 ; http://php.net/allow-url-fopen
-allow_url_fopen = On
+allow_url_fopen = Off
 
 ; Whether to allow include/require to open URLs (like http:// or ftp://) as files.
 ; http://php.net/allow-url-include
@@ -897,8 +897,8 @@
 ;extension=php_pspell.dll
 ;extension=php_shmop.dll
 
-; The MIBS data available in the PHP distribution must be installed. 
-; See http://www.php.net/manual/en/snmp.installation.php 
+; The MIBS data available in the PHP distribution must be installed.
+; See http://www.php.net/manual/en/snmp.installation.php
 ;extension=php_snmp.dll
 
 ;extension=php_soap.dll
@@ -920,7 +920,7 @@
 [Date]
 ; Defines the default timezone used by the date functions
 ; http://php.net/date.timezone
-;date.timezone =
+date.timezone = PRC
 
 ; http://php.net/date.default-latitude
 ;date.default_latitude = 31.7667
@@ -1007,23 +1007,23 @@
 
 ; For Win32 only.
 ; http://php.net/sendmail-from
-;sendmail_from = me@example.com
+sendmail_from = noreply@phpvim.me
 
 ; For Unix only.  You may supply arguments as well (default: "sendmail -t -i").
 ; http://php.net/sendmail-path
-;sendmail_path =
+sendmail_path = /usr/bin/sendmail -t -i
 
 ; Force the addition of the specified parameters to be passed as extra parameters
 ; to the sendmail binary. These parameters will always replace the value of
 ; the 5th parameter to mail(), even in safe mode.
-;mail.force_extra_parameters =
+mail.force_extra_parameters =
 
 ; Add X-PHP-Originating-Script: that will include uid of the script followed by the filename
 mail.add_x_header = On
 
 ; The path to a log file that will log all mail() calls. Log entries include
 ; the full path of the script, line number, To address and headers.
-;mail.log =
+mail.log = /var/log/php/php-mail.log
 ; Log mail to syslog (Event Log on NT, not valid in Windows 95).
 ;mail.log = syslog
 
@@ -1397,7 +1397,7 @@
 ; where MODE is the octal representation of the mode. Note that this
 ; does not overwrite the process's umask.
 ; http://php.net/session.save-path
-;session.save_path = "/tmp"
+session.save_path = /var/lib/php/session
 
 ; Whether to use strict session mode.
 ; Strict session mode does not accept uninitialized session ID and regenerate
@@ -1591,7 +1591,7 @@
 ; Development Value: On
 ; Production Value: On
 ; http://php.net/session.upload-progress.enabled
-;session.upload_progress.enabled = On
+session.upload_progress.enabled = On
 
 ; Cleanup the progress information as soon as all POST data has been read
 ; (i.e. upload completed).
@@ -1599,14 +1599,14 @@
 ; Development Value: On
 ; Production Value: On
 ; http://php.net/session.upload-progress.cleanup
-;session.upload_progress.cleanup = On
+session.upload_progress.cleanup = On
 
 ; A prefix used for the upload progress key in $_SESSION
 ; Default Value: "upload_progress_"
 ; Development Value: "upload_progress_"
 ; Production Value: "upload_progress_"
 ; http://php.net/session.upload-progress.prefix
-;session.upload_progress.prefix = "upload_progress_"
+session.upload_progress.prefix = "upload_progress_"
 
 ; The index name (concatenated with the prefix) in $_SESSION
 ; containing the upload progress information
@@ -1614,7 +1614,7 @@
 ; Development Value: "PHP_SESSION_UPLOAD_PROGRESS"
 ; Production Value: "PHP_SESSION_UPLOAD_PROGRESS"
 ; http://php.net/session.upload-progress.name
-;session.upload_progress.name = "PHP_SESSION_UPLOAD_PROGRESS"
+session.upload_progress.name = "PHP_SESSION_UPLOAD_PROGRESS"
 
 ; How frequently the upload progress should be updated.
 ; Given either in percentages (per-file), or in bytes
@@ -1622,14 +1622,14 @@
 ; Development Value: "1%"
 ; Production Value: "1%"
 ; http://php.net/session.upload-progress.freq
-;session.upload_progress.freq =  "1%"
+session.upload_progress.freq =  "1%"
 
 ; The minimum delay between updates, in seconds
 ; Default Value: 1
 ; Development Value: 1
 ; Production Value: 1
 ; http://php.net/session.upload-progress.min-freq
-;session.upload_progress.min_freq = "1"
+session.upload_progress.min_freq = "1"
 
 [MSSQL]
 ; Allow or prevent persistent links.
@@ -1907,7 +1907,7 @@
 ;opcache.revalidate_path=0
 
 ; If disabled, all PHPDoc comments are dropped from the code to reduce the
- ;size of the optimized code.
+; size of the optimized code.
 ;opcache.save_comments=1
 
 ; If disabled, PHPDoc comments are not loaded from SHM, so "Doc Comments"
