--- sapi/fpm/php-fpm.service.in.old	2013-11-15 13:53:32.870050573 +0800
+++ sapi/fpm/php-fpm.service.in	2013-11-15 13:55:56.960052806 +0800
@@ -1,12 +1,12 @@
 [Unit]
 Description=The PHP FastCGI Process Manager
-After=syslog.target network.target
 
 [Service]
-Type=@php_fpm_systemd@
-PIDFile=@localstatedir@/run/php-fpm.pid
-ExecStart=@sbindir@/php-fpm --nodaemonize --fpm-config @sysconfdir@/php-fpm.conf
+Type=forking
+PIDFile=/run/php-fpm.pid
+ExecStart=@sbindir@/php-fpm --daemonize --fpm-config @sysconfdir@/php-fpm.conf
 ExecReload=/bin/kill -USR2 $MAINPID
+ExecStop=/bin/kill $MAINPID
 
 [Install]
 WantedBy=multi-user.target
